app/src/main/java/org/imozerov/streetartview/storage/DataSource.kt

```diff
@@ -73,19 +69,28 @@ class DataSource(private val realm: Realm, private val handler: Handler) {
         with (realmAuthor) {
             id = SystemClock.currentThreadTimeMillis().toString()
             name = "${randomFrom(names)} ${randomFrom(lastNames)}"
-            description = "The best artist in the world"
+            photo = "The best artist in the world"
+        }
+
+        val realmAuthors = RealmList<RealmAuthor>()
+        realmAuthors.add(realmAuthor)
+
+        val realmLocation = RealmLocation()
+        with(realmLocation) {
+            address = "Some address, 34"
+            lat = getRandomBetween(56.26, 56.33)
+            lng = getRandomBetween(43.86, 44.05)
         }
 
         val realmArtObject = RealmArtObject()
         with (realmArtObject) {
-            author = realmAuthor
+            authors = realmAuthors
```

Comment:
Does it mean that now several authors can correspond to an art object in our model?